programs/session-manager/src/intents/message.rs

```diff
@@ -4,6 +4,7 @@ use crate::{
 };
 use anchor_lang::prelude::*;
 use chrono::DateTime;
+use rust_decimal::{dec, Decimal};
```

Comment:
this import is only used in tests, can you move it to the test module?

Reply:
yep, good call, will fix